test/ParallelIOTest.cpp

```diff
@@ -422,6 +422,48 @@ void available_chunks_test(std::string const &file_ending)
         auto E_x = it0.meshes["E"]["x"];
         E_x.resetDataset({Datatype::INT, {mpi_size, 4}});
         E_x.storeChunk(data, {mpi_rank, 0}, {1, 4});
+
+        auto electrons = it0.particles["e"].particlePatches;
+        auto numParticles = electrons["numParticles"];
+        auto numParticlesOffset = electrons["numParticlesOffset"];
```

Comment:
Do you want to add an inline comment here somewhere what the block is functionally testing? :)